MSMetaEnhancer/libs/converters/web/IDSM.py

```diff
@@ -120,6 +126,7 @@ async def from_name(self, name):
         """
         return await self.call_service(query)
 
+    @escape_single_quotes
     async def from_inchi(self, inchi):
         """
         Convert InChi to to all possible attributes using IDSM service
```

Comment:
What is this notation for function calls? How is this different from calling a function normally?